specs-go/config.go

```diff
@@ -430,6 +430,8 @@ type SolarisAnet struct {
 
 // Windows defines the runtime configuration for Windows based containers, including Hyper-V containers.
 type Windows struct {
+	// LayerFolders contains a list of absolute paths to directories containing image layers.
+	LayerFolders []string `json:"layerFolders"`
```

Comment:
> Should be all lower-case for consistency.

There are currently some camelCase entries in `config-windows.md`.  For example, [the `network` properties][1].

[1]: https://github.com/opencontainers/runtime-spec/blame/2ec18c02474dd49ce20aa30236959a8a80763042/config-windows.md#L84-L87